src/views/blocks/rsvp/form.php

```diff
@@ -22,11 +24,12 @@
 <div class="tribe-block__rsvp__form">
 	<?php if ( ! empty( $going ) && ! $must_login ) :
 		$ticket = $this->get( 'ticket' );
-		$args = array(
-					'ticket_id' => $ticket->ID,
-					'ticket'    => tribe( 'tickets.rsvp' )->get_ticket( get_the_id(), $ticket->ID ),
-					'going'     => esc_html( $going ),
-				);
+		$args = [
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: 367996643fd173e404c32bd8e50859c1</sub>